client/signup/steps/survey/index.jsx

```diff
@@ -59,12 +60,13 @@ export default React.createClass( {
 		const stepOneClickHandler = ( event ) => {
 			event.preventDefault();
 			event.stopPropagation();
+			this.setState( { surveyFirstQuestion: vertical.value } );
```

Comment:
I haven't completely grokked the changes to this file, but setting state inside a render method is a little concerning, mainly because it could lead to infinite loops (`setState` calls `render`, `render` calls `setState`). If you're just looking to record the vertical ID and pass it along, we already have that data inside `handleNextStep` (if configured to show two steps, which we are, the first question value is in `this.state.stepOne.value`).